packages/sshnoports/lib/sshnp/sshnp_arg.dart

```diff
@@ -195,3 +214,47 @@ class SSHNPArg {
     return 'SSHNPArg{format: $format, name: $name, abbr: $abbr, help: $help, mandatory: $mandatory, defaultsTo: $defaultsTo, type: $type}';
   }
 }
+
+ArgParser createArgParser({
```

Comment:
Moved this from SSHNPParams so that SSHNPParams would be more focused in intention